api/services/gameService.js

```diff
@@ -177,7 +178,8 @@ module.exports = {
         ...gameUpdates,
       };
       if (game.isRanked) {
-        await sails.helpers.addGameToMatch(game);
+        const currentMatch = await sails.helpers.addGameToMatch(game);
+        res.currentMatch = currentMatch;
```

Comment:
```suggestion
        res.currentMatch = await sails.helpers.addGameToMatch(game);
```
Doesn't seem to be reused below.